### PR TITLE
Miscellaneous installer overhaul improvements found during testing/feedback

### PR DESCRIPTION
**Pull request by jjrush**
- Small fixes to make traefik.label test pass - always write a value to the file
- Moving some items from boolean to complex and adding custom transforms for them
- Updating netsniff and arkime dependencies
- Removing unused complex dependencies code
- Removing low value tests and improving .env roundtrip, edge case, and sentinel value tests

### CODE REVIEW

File: scripts/installer/actions/shared.py [hunk 1]
```diff
@@ -398,7 +402,9 @@ def _apply_exposed_services(data: dict, exposed_services_tuple, platform) -> Non
             data['services']['nginx-proxy'].pop('ports', None)
         else:
             data['services']['nginx-proxy']['ports'] = [
-                f"{':'.join(SERVICE_IP_EXPOSED, SERVICE_PORT_MALCOLM) if nginx_ssl else ':'.join(SERVICE_IP_LOCAL, SERVICE_PORT_MALCOLM_NO_SSL)}:{SERVICE_PORT_MALCOLM}/tcp",
+                f"{(SERVICE_IP_EXPOSED if nginx_ssl else SERVICE_IP_LOCAL)}:"
+                f"{(SERVICE_PORT_MALCOLM if nginx_ssl else SERVICE_PORT_MALCOLM_NO_SSL)}:"
+                f"{SERVICE_PORT_MALCOLM}/tcp",
```
Comment (jjrush) [inline scripts/installer/actions/shared.py]: There was a typo in the original and this was just cleaner to read. You can concatenate it back into a single line if you like.

File: scripts/installer/actions/shared.py [hunk 2]
```diff
@@ -520,7 +524,7 @@ def update_compose_files(
                 finally:
                     # restore ownership
                     if written:
-                        os.chown(config_file_stat, orig_uid, orig_gid)
+                        os.chown(config_file, orig_uid, orig_gid)
```
Comment (jjrush) [inline scripts/installer/actions/shared.py]: This was failing since we weren't `chown`ing the actual file.
